docker/Dockerfile

```diff
@@ -0,0 +1,130 @@
+# Handle ros distro
+ARG ROS_DISTRO=noetic
+FROM ghcr.io/aica-technology/ros-ws:${ROS_DISTRO}
+
+# User provided arguments
+ARG HOST_GID=1000
+ARG GIT_NAME=""
+ARG GIT_EMAIL=""
+ARG USE_SIMD=OFF
```

Comment:
FYI: My understanding is that it creates the wrong impression that this is still an option.
The dependencies such as RBDyn are currently always without SIMD, so setting this true would result in crashes